### PR TITLE
Fix Focus styling for AreaChart Basic Example in docsite

### LINKED ISSUE #30853: [AreaChart-AreaChart basic]: Keyboard focus indicators not visible on 'Change width' and 'Change height' sliders in 'Area chart basic' section when navigating with tab key

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior
Focus styling for AreaChart Basic example was not working (overridden by global styling)

## New Behavior
Focus styling fixed by custom styling.
image
image

## Related Issue(s)

- Fixes #30853

### CODE REVIEW

File: packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ interface IAreaChartBasicState {
   height: number;
   isCalloutselected: boolean;
   showAxisTitles: boolean;
+  isHeightSliderFocused: boolean;
+  isWidthSliderFocused: boolean;
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx:13]: Focus bar is not showing anywhere on the graph. Not only for width and height sliders

File: packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx [hunk 2]
```diff
@@ -26,9 +26,26 @@ export class AreaChartBasicExample extends React.Component<{}, IAreaChartBasicSt
...
+      outline: -webkit-focus-ring-color auto 5px;
+    }
+    `;
+    style.appendChild(document.createTextNode(focusStylingCSS));
+    document.head.appendChild(style);
+  }
 
   public render(): JSX.Element {
-    return <div>{this._basicExample()}</div>;
+    return <div className="rootDiv">{this._basicExample()}</div>;
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx:48]: call this containerDiv.

Comment (Shubhabrata08) [reply packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx]: Done

**Review by AtishayMsft — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Pull request merged**